src/pages/User/components/PullRequests/index.js

```diff
@@ -88,6 +90,8 @@ class PullRequests extends Component {
       );
 
       const [data, userDetail] = await Promise.all(allResponses);
+
+      data.items = data.items.filter(pullRequest => isPRLabelValid(pullRequest.labels));
```

Comment:
Let's try and avoid mutations like this. Instead `setState` the filtered array in line `98`.